packages/nextjs/services/database/seed.data.example.ts

```diff
@@ -364,25 +361,25 @@ export const seedUserChallenges: (typeof userChallenges.$inferInsert)[] = [
   },
 ];
 
-export const seedBatches: (typeof batches.$inferInsert)[] = [
+// userAddresses denotes the user addresses that will be added to the batch
+export const seedBatches: (typeof batches.$inferInsert & { userAddresses?: string[] })[] = [
   {
-    id: 1,
     name: "Batch 1",
     startDate: new Date(1705315180000),
     status: BatchStatus.CLOSED,
     telegramLink: "https://t.me/joinchat/1",
     contractAddress: "0x0000000000000000000000000000000000000000",
+    userAddresses: [seedUsers?.[0]?.userAddress, seedUsers?.[1]?.userAddress],
   },
   {
-    id: 2,
     name: "Batch 2",
     startDate: new Date(1707820780000),
     status: BatchStatus.CLOSED,
     telegramLink: "https://t.me/joinchat/2",
     contractAddress: "0x0000000000000000000000000000000000000000",
+    userAddresses: [seedUsers?.[2]?.userAddress, seedUsers?.[3]?.userAddress],
```

Comment:
Can you add `batchStatus` to `seedUser[3]`? It's `null` now.

I guess this is something we could fix on the database too (batch status should probably be notNull and default to candidate)

Reply:
Ohh nice catch! Thanks updated it at https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/143/commits/0308c1fdcd03ebbb706af1499e2b57ba96e75284

Reply:
> Like somehow on db level we need to make sure if batch_id exist then batch_status shouldn't be null at all.

Not sure if it's possible to enforce this on the database level, but yes, that would be what we need. No big deal tho!